api/src/test/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactoryTest.java

```diff
@@ -205,6 +244,27 @@ private ReportClaimMessageContext aReportClaimMessageContext(LocalDateTime times
                 .build();
     }
 
+    private ReportPaymentMessageContext aReportPaymentMessageContext(LocalDateTime timestamp,
```

Comment:
Worth adding to a test data factory?

Reply:
I won't for now as it it's just used here. Perhaps in a future PR when more of these are created, need to see how much the test objects vary.